testdata/grpc-web-client-impl-config.yaml

```diff
@@ -0,0 +1,13 @@
+features:
+  versions:
+    - HTTP_VERSION_1
+  protocols:
+    - PROTOCOL_GRPC_WEB
+  codecs:
+    - CODEC_PROTO
+  compressions:
+    - COMPRESSION_IDENTITY
```

Comment:
From above thread: this should include gzip if the grpcweb client can handle compression. Maybe a TODO for now (you can put TODOs in this file -- it's yaml, so comments are allowed).